native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java

```diff
@@ -428,6 +437,20 @@ private static PrimitiveTypeColumnDefinition generateColumnDefinition(SQLColumnM
                 ballerinaType);
     }
 
+    private static String getColumnName(StructureType streamConstraint, String fieldName) {
```

Comment:
Shall we rename it to `getAnnotatedColumnName`?

Reply:
Fixed in [f776db6](https://github.com/ballerina-platform/module-ballerina-sql/pull/509/commits/f776db65ea7ce37148205e201b8c7d69ca7e0049)